recipe/meta.yaml

```diff
@@ -38,13 +38,14 @@ test:
     - ${INSTALL_NAME_TOOL} -change @rpath/libtinyxml2.{{ version.split('.')[0] }}.dylib $CONDA_PREFIX/lib/libtinyxml2.{{ version.split('.')[0] }}.dylib xmltest  # [osx]
     - ./xmltest  # [unix or osx]
 
-    - if exist %PREFIX%\\Library\\lib\\cmake\\tinyxml2\\tinyxml2Config.cmake (exit 0) else (exit 1)  # [win]
-    - if exist %PREFIX%\\Library\\lib\\cmake\\tinyxml2\\tinyxml2Targets.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %PREFIX%\\Library\\bin\\tinyxml2.dll (exit 0) else (exit 1)  # [win]
```

Comment:
@wolfv It looks like the build [no longer supports both shared and static builds at the same time](https://github.com/leethomason/tinyxml2/blob/8.1.0/CMakeLists.txt#L85). Do you think the static library is needed?